scanners/trivy/examples/k8s/scan.yaml

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: "execution.securecodebox.io/v1"
+kind: Scan
+metadata:
+  name: "trivy-k8s"
+spec:  
+  scanType: "trivy-k8s"
+  parameters:
+    - "cluster"
```

Comment:
Is there maybe an easy way to also add an integration-test for trivy-k8s scantype ? That would help alot in supporting it :)

Reply:
integration test added. only structure of result-document is assured in test.